src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Updated to use Activity native support from `System.Diagnostics.DiagnosticSource`
```

Comment:
```suggestion
* Use `Activity.Status` and `Activity.StatusDescription` properties instead of
  `OpenTelemetry.Trace.Status` and `OpenTelemetry.Trace.Status.Description`
  respectively.
```